src/main/resources/templates/start.html

```diff
@@ -15,7 +15,7 @@
 	    <link type="text/css" rel="stylesheet" href="/Enterprise-App-Project/src/main/resources/templates/start.css" />
 	    <style>
 	    .intro {
-	    	border-color: 9fa0ff;
+	    	border-color: #9fa0ff;
```

Comment:
Some small issues with html and css - color hex codes need to be prefixed with a #, and html attributes need a =.